### PR TITLE
minimal link-type implementation

### PR DESCRIPTION
**Pull request by simontegg**

### CODE REVIEW

File: src/node_modules/link-type/model.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// var debug = require('debug')('holodex:link-type:model')
+
+var Model = require('base/model')
+
+module.exports = Model.extend({
+  typeName: 'LinkType',
+  // collectionName: 'linkTypes',
+
```
Comment (simontegg) [inline src/node_modules/link-type/model.js:8]: don't know if this needs a collection or other props?

**Comment by simontegg** [comment]
ping @ahdinosaur

File: src/node_modules/link-type/model.js (hunk 1 above)
Comment (ahdinosaur) [inline src/node_modules/link-type/model.js:8]: i think it's good as is.

**Comment by ahdinosaur** [comment]
looks good to me. :+1:

**Comment by ahdinosaur** [comment]
oops, silly me forgot to deploy this before merging https://github.com/holodex/enspiral-data/pull/19. all good now. :deciduous_tree:

**Pull request merged**